lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -67,7 +67,6 @@
                     // name is being routed to.
                     router = this.router = new Backbone.Router();
                     _.each([
-                        [':default', _.bind(this.routeNotFound, this)],
```

Comment:
Got it, thanks. Why does Backbone required "hijacking" though? Seems kinda... weird.